platform/platform-configuration-listener/src/test/java/org/codice/ddf/configuration/store/SystemConfigurationMigratorTest.java

```diff
@@ -0,0 +1,336 @@
+                + VALID_CRL_REL_PATH_1));
+        assertThat(sourceFiles.get(3).getAbsolutePath(), is(DDF_HOME + File.separator
+                + SYSTEM_PROPERTIES_REL_PATH));
+        assertThat(destinationFiles.get(3).getAbsolutePath(), is(EXPORT_DIRECTORY + File.separator
+                + SYSTEM_PROPERTIES_REL_PATH));
+        assertThat(sourceFiles.get(4).getAbsolutePath(), is(DDF_HOME + File.separator
+                + USERS_PROPERTIES_REL_PATH));
+        assertThat(destinationFiles.get(4).getAbsolutePath(), is(EXPORT_DIRECTORY + File.separator
+                + USERS_PROPERTIES_REL_PATH));
+    }
+
+    @Test
+    public void testExportValidAbsolutePaths() throws Exception {
+        // Setup
+        when(System.getProperty(KEYSTORE_SYSTEM_PROP)).thenReturn(KEYSTORE_ABS_PATH);
+        when(System.getProperty(TRUSTSTORE_SYSTEM_PROP)).thenReturn(TRUSTSTORE_ABS_PATH);
+        Stream<String> stream = getStream(ENC_PROP_FILE_ABS_CRL_PATH);
+        doNothing().when(FileUtils.class);
+        SystemConfigurationMigrator securityConfigurationMigrator = new SystemConfigurationMigrator(
+                DDF_HOME) {
+            @Override
+            Stream<String> getStream(Path path) throws IOException {
+                return stream;
+            }
+        };
+
+        // Perform Test
+        securityConfigurationMigrator.export(EXPORT_DIRECTORY);
+
+        // Verify
+        ArgumentCaptor<File> sourceDirCaptor = ArgumentCaptor.forClass(File.class);
+        ArgumentCaptor<File> destinationDirCaptor = ArgumentCaptor.forClass(File.class);
+        verifyStatic(times(2));
+        FileUtils.copyDirectory(sourceDirCaptor.capture(), destinationDirCaptor.capture());
+        List<File> sourceDirs = sourceDirCaptor.getAllValues();
+        List<File> destinationDirs = destinationDirCaptor.getAllValues();
+        assertThat(sourceDirs.get(0).getAbsolutePath(), is(DDF_HOME + File.separator
+                + WS_SECURITY_DIR_REL_PATH));
+        assertThat(destinationDirs.get(0).getAbsolutePath(), is(EXPORT_DIRECTORY + File.separator
+                + WS_SECURITY_DIR_REL_PATH));
+        assertThat(sourceDirs.get(1).getAbsolutePath(), is(DDF_HOME + File.separator
+                + PDP_POLICIES_DIR_REL_PATH));
+        assertThat(destinationDirs.get(1).getAbsolutePath(), is(EXPORT_DIRECTORY + File.separator
+                + PDP_POLICIES_DIR_REL_PATH));
+
+        ArgumentCaptor<File> sourceFileCaptor = ArgumentCaptor.forClass(File.class);
+        ArgumentCaptor<File> destinationFileCaptor = ArgumentCaptor.forClass(File.class);
+        List<File> sourceFiles = sourceFileCaptor.getAllValues();
+        List<File> destinationFiles = destinationFileCaptor.getAllValues();
+        verifyStatic(times(5));
+        FileUtils.copyFile(sourceFileCaptor.capture(), destinationFileCaptor.capture());
+        assertThat(sourceFiles.get(0).getAbsolutePath(), is(DDF_HOME + File.separator
```

Comment:
If I'm not mistaken, those assertions and the ones above should be identical to the ones in `testExportValidRelativePaths()` so it'd be nice if we could extract a couple private methods for this.